maquetado/index.html

```diff
@@ -1,14 +1,22 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
+
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="container fields-component column horizontally-centered vertically-space-between">
+    <h1 class="title">T&iacute;tulo del libro <span class="field-description strong">(g&eacute;nero)</span></h1>
+    <p class="field" style="margin-left: 55px">Autor del libro: <span class="field-description">Nombre del autor del libro</span></p>
```

Comment:
dont use inline styles, try to use classes.. I don't understand why all the p elements have different margins

Reply:
I do that in order to put all labels in the same begging line as the title. Since there are centered I have to manipulate the padding to make them appear in the same line as the title